From d7c0b0f04f20b0aa57fbf05dee7374b7558f9051 Mon Sep 17 00:00:00 2001
From: Thomas Roos <throos@amazon.de>
Date: Wed, 23 Jul 2025 16:39:01 +0000
Subject: [PATCH] amazon-kvs-webrtc-sdk: disable download of kvs-common-lws

This will disable the download of the kvsCommonLws library, which is not needed for the build.
As we use the system-installed version of kvsCommonLws, we do not need to download it again.

Upstream-Status: Inappropriate [oe-specific]
---
 CMakeLists.txt | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 51f67057b7..c839edc723 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -265,16 +265,6 @@ if(BUILD_DEPENDENCIES)
   message(STATUS "Finished building dependencies.")
 endif()
 
-# building kvsCommonLws also builds kvspic
-set(BUILD_ARGS
-        -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
-        -DBUILD_STATIC=${BUILD_STATIC_LIBS}
-        -DUSE_OPENSSL=${USE_OPENSSL}
-        -DUSE_MBEDTLS=${USE_MBEDTLS}
-        -DKVS_DEFAULT_STACK_SIZE=${KVS_DEFAULT_STACK_SIZE}
-        -DCMAKE_C_FLAGS=${CMAKE_C_FLAGS})
-build_dependency(kvsCommonLws ${BUILD_ARGS})
-
 ############# find dependent libraries ############
 
 find_package(Threads)
